TdsModel.h

```diff
@@ -297,6 +297,9 @@ class TdsModel
   void getSectionFlow(int sectionIndex, double &inflow, double &outflow);
   double getSectionPressure(int sectionIndex);
 
+  void calcNoiseSample(NoiseSource* s, double ampThreshold);
+  void incrementPosition() { position++; }
```

Comment:
Is this function needed?

Reply:
Same as before, this was to use the noise function.

Reply:
Got it, so we can just remove it, then.